ara/cli/host.py

```diff
@@ -76,15 +76,28 @@ class HostShow(ShowOne):
     def get_parser(self, prog_name):
         parser = super(HostShow, self).get_parser(prog_name)
         parser.add_argument(
+            '--playbook', '-b',
+            metavar='<playbook-id>',
+            help='Show host associated with the given playbook',
+        )
+        parser.add_argument(
             'host',
             metavar='<host>',
-            help='Host id to show',
+            help='Host name (when using --playbook) or id to show',
```

Comment:
I think this could be better phrased "Host id (or name when using --playbook) to show".